_health-care/_js/components/form-elements/GrowableTable.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+
+class GrowableTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleAdd = this.handleAdd.bind(this);
+  }
+
+  handleAdd() {
+    const blankRowData = this.props.createRow();
+    const rows = this.props.rows.slice();
+    rows.push(blankRowData);
+    this.props.onRowsUpdate(rows);
+  }
+
+  render() {
+    const component = this.props.component;
+    const rowElements = this.props.rows.map((obj, index) => {
+      return (
+        <div className="row" key={`row-${index}`}>
+          <hr/>
+          <div className="small-3 columns" key={`label-${index}`}>{index}</div>
```

Comment:
Maybe we should display `{index + 1}`, since it starts at 0, which is confusing :)